keyboards/zsa/moonlander/moonlander.c

```diff
@@ -91,6 +94,7 @@ static THD_FUNCTION(LEDThread, arg) {
     chRegSetThreadName("LEDThread");
     while (true) {
         moonlander_led_task();
+        wait_ms(100);
```

Comment:
I'd almost opt for this being configurable. 
Eg, add something like towards the top: 
```c
#ifndef MOONLANDER_TREAD_INTERVAL_MS
#    define MOONLANDER_TREAD_INTERVAL_MS 100
#endif // MOONLANDER_TREAD_INTERVAL_MS
```
And set this to: 
```suggestion
        wait_ms(MOONLANDER_TREAD_INTERVAL_MS);
```

Reply:
Done.